packages/contracts/deploy/30_plugins/10_plugin-repos/00_create-plugins-repo.ts

```diff
@@ -1,7 +1,12 @@
 import {HardhatRuntimeEnvironment} from 'hardhat/types';
 import {DeployFunction} from 'hardhat-deploy/types';
 
-import {createPluginRepo} from '../../helpers';
+import addresslistMetadata from '../../../contracts/voting/addresslist/metadata.json';
```

Comment:
Please, let's double check the task description

We need plugins to have two different "metadata's":
- Per plugin (global)
  - Name
  - Description
  - Screenshots, etc
- Per version
  - Changes
  - ABI's

Reply:
Let's do `per plugin's release` and not `per plugin`. Thoughts ? @brickpop as each plugin release might have different pictures/information.